src/index.js

```diff
@@ -0,0 +1,23 @@
+/*eslint-disable*/
+import _ from 'lodash';
+/* eslint-enable */
+import './style.css';
+import Scores from './scores';
+
+function updateScore() {
+  const myScores = new Scores();
+  let i = 0;
+  const scores = document.getElementById('scores');
+  scores.classList.add('black-border');
+  myScores.scoreList.forEach((score) => {
+    const li = document.createElement('li');
+    if (i % 2 === 0) {
```

Comment:
- [ ] From the learning objectives, you are encouraged to use ES6 syntax and this includes replacing the `function` keyword with `arrow functions`. Kindly work on this